spec/factories/comments.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :comment do
+    content "MyText"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.